rise-playlist.html

```diff
@@ -254,7 +254,7 @@
        */
       hide: function() {
         for (var i = 0; i < this.items.length; i++) {
-          this.items[i].element.style.visibility = "hidden";
+          this.items[i].element.style.display = "none";
```

Comment:
@donnapep Not disagreeing with this change, but just curious about what made you set `display = "none"` instead of changing visibility?